views/av-evidence-tab.html

```diff
@@ -26,7 +26,7 @@ <h1 class="govuk-heading-xl">{{ content[i18n.language].benefitTypes[appeal.benef
                 <h2 class="govuk-heading-m --margin-bottom-s">{{ content[i18n.language].common.avEvidence | safe }}</h2>
                 <br/>
                 <div>
-                    {% if not avEvidenceList %}
+                    {% if not appeal.audioVideoEvidence %}
```

Comment:
I spotted this line of code inside the project (  let avEvidenceList = session['appeal'].audioVideoEvidence; ), Does this not set audioVideoEvidence for avEvidenceList? I would double check this please